towhee/engine/operator_io/reader.py

```diff
@@ -136,3 +136,105 @@ def read(self) -> List[Dict[str, any]]:
     def close(self):
         self._close = True
         self._iter.notify()
+
+
+class _TimeWindow:
+    '''
+    '''
+
+    def __init__(self, time_range_sec: int, time_step_sec: int, start_time_sec: int = 0):
+        self._start_time_m = start_time_sec * 1000
```

Comment:
Why are the seconds multiplied by 1000 throughout the code?

Reply:
The timestamp of video frame is millisecond.